### DISCUSSION TITLE
weeklies/1

### DISCUSSION THREAD

**Discussion by hutusi** [hutusi/hutusi.github.com#146] [Comments]
https://hutusi.com/articles/weekly-1 

 阮一峰在两年前开始做周刊，记录每周值得分享的科技内容，目前已经做到99期了。受他的启发，我也尝试写写周刊，一是总结自己的一周或几周所看所学，二是做个记录和分享。为了避免重蹈先前每天一篇公众号文章的覆辙，我不会限定每周一刊，可能两周，也可能四周，视情形而定。

**Comment by petite-dd** [comment]
非常优秀，向你学习

**Comment by hutusi** [comment]
> @petite-dd
> 非常优秀，向你学习

谢谢你的鼓励:-)
